terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -178,7 +178,7 @@ module "server_containerized" {
   main_disk_size = 3000
   runtime = "podman"
   # Temporary using the correct URL for the 5.0.1 BV only. See https://github.com/SUSE/spacewalk/issues/25120
-  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile"
```

Comment:
SUMA containers are behind  login, but  this  is not the case  for Uyuni.
you can't align,  unless you  can  make  sure that you can pick the right namespace that includes the architecture

Reply:
in addition to this, I see  you use the r`egistry.suse.de/suse/maintenance/35237`. I suppose this tf file is for validating 5.0.1 and not in general  5.0, right?

Reply:
5.0 is SUMA, not Uyuni. Why are you mentioning Uyuni ?

Reply:
yes, next one will be 5.0.1

maintenance/35237 is marked as "temporary" and i suppose it will go... maybe i should have removed it already ? i'll do that

Reply:
done